[2022-08-29T12:34:56.583-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 scheduled__2022-08-28T00:00:00+00:00 [queued]>
[2022-08-29T12:34:56.598-0400] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 scheduled__2022-08-28T00:00:00+00:00 [queued]>
[2022-08-29T12:34:56.598-0400] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T12:34:56.598-0400] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-08-29T12:34:56.598-0400] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-08-29T12:34:56.645-0400] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): runme_0> on 2022-08-28 00:00:00+00:00
[2022-08-29T12:34:56.647-0400] {standard_task_runner.py:52} INFO - Started process 512 to run task
[2022-08-29T12:34:56.652-0400] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_0', 'scheduled__2022-08-28T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', '/home/augenz/PycharmProjects/airflow/.env_airflow/lib/python3.9/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmphr8rsy7r', '--error-file', '/run/user/1000/app/com.jetbrains.PyCharm-Community/tmpkl1fw6he']
[2022-08-29T12:34:56.653-0400] {standard_task_runner.py:80} INFO - Job 2: Subtask runme_0
[2022-08-29T12:34:56.829-0400] {task_command.py:371} INFO - Running <TaskInstance: example_bash_operator.runme_0 scheduled__2022-08-28T00:00:00+00:00 [running]> on host pop-os.localdomain
[2022-08-29T12:34:57.042-0400] {taskinstance.py:1581} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_0
AIRFLOW_CTX_EXECUTION_DATE=2022-08-28T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-28T00:00:00+00:00
[2022-08-29T12:34:57.044-0400] {subprocess.py:62} INFO - Tmp dir root location: 
 /run/user/1000/app/com.jetbrains.PyCharm-Community
[2022-08-29T12:34:57.045-0400] {subprocess.py:74} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "example_bash_operator__runme_0__20220828" && sleep 1']
[2022-08-29T12:34:57.053-0400] {subprocess.py:85} INFO - Output:
[2022-08-29T12:34:57.057-0400] {subprocess.py:92} INFO - example_bash_operator__runme_0__20220828
[2022-08-29T12:34:58.089-0400] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-29T12:34:58.205-0400] {taskinstance.py:1407} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_0, execution_date=20220828T000000, start_date=20220829T163456, end_date=20220829T163458
[2022-08-29T12:34:58.271-0400] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-29T12:34:58.436-0400] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
